index.html

```diff
@@ -2423,26 +2423,25 @@ <h2>Contexts and Vocabularies</h2>
         <h3>Context Injection</h3>
 
         <p>
-The `@context` property is used by implementations that employ JSON-LD to ensure that implementations are using the
+The `@context` property is used to ensure that implementations are using the
 same semantics when terms in this specification are processed. For example, this
-can be important when properties like `type` are processed and its values, such
-as `DataIntegrityProof`, are used.
+can be important when properties like `authentication` are processed and its
+value, such as `Multikey` or `JsonWebKey`, are used.
         </p>
 
         <p>
-If an `@context` property is not provided in a document that is being secured or
-verified, or any Data Integrity terms used in the document are not mapped by
-existing values in the `@context` property, implementations using JSON-LD MUST inject or add
-an `@context` property with a value of
-`https://www.w3.org/ns/controller/v1`.
+When an application is processing a [=controller document=], if an `@context`
+property is not provided in the document or the terms used in the document are
+not mapped by existing values in the `@context` property, implementations aware
+of JSON-LD MUST inject or append an `@context` property with a value of
+`https://www.w3.org/ns/controller/v1` or one or more contexts with at least the
+same declarations, such as the Decentralized Identifier v1.1 context
+(`https://www.w3.org/ns/did/v1.1`).
         </p>
 
         <p>
-Context injection is expected to be unnecessary sometimes, such as when the Verifiable
-Credential Data Model v2.0 context (`https://www.w3.org/ns/credentials/v2`)
-exists as a value in the `@context` property, as that context maps all of the
-necessary Data Integrity terms that were previously mapped by
-`https://w3id.org/security/data-integrity/v2`.
+Implementations that do not intend to use JSON-LD MAY choose to not include an
+`@context` declaration at the top-level of the document.
         </p>
```

Comment:
There should be another sentence here to indicate what the consequences are of not choosing to include an 
`@context` declaration in the document

Reply:
@David-Chadwick and @TallTed: please, propose the change in #92 (which has taken over this PR, and that context injection part is identical to this PR). Thx.